hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaAction.java

```diff
@@ -137,6 +137,9 @@
     @Column(name = "external_ref", length = Action.EXTERNAL_REF_MAX_LENGTH)
     private String externalRef;
 
+    @Column(name = "initiated_by", updatable = false, nullable = false, length = 64)
```

Comment:
If you set this field as non-nullable, what happens to already existing actions in the DB, is a migration script required? Could you also define the length of this field in `Action` interface

Reply:
no migration needed for old actions, since this value is not updatable
I also tested it with the following code path, works without issues
```java
@Bean
ApplicationRunner testOldActions(final TenantAware tenantAware, final ActionRepository actionRepository) {
        return args -> {
            tenantAware.runAsTenant("DEFAULT", () -> {
                actionRepository.findAll().forEach(a -> {
                    if (a.isActive()) {
                        a.setActionType(Action.ActionType.DOWNLOAD_ONLY);
                        actionRepository.save(a);
                    }
                });
                return null;
            });
        };
    }
```